web/ui/react-app/src/pages/graph/Panel.tsx

```diff
@@ -117,21 +118,20 @@ class Panel extends Component<PanelProps & PathPrefixProps, PanelState> {
     let path: string;
     switch (this.props.options.type) {
       case 'graph':
-        path = '/api/v1/query_range';
+        path = `query_range`;
         params.append('start', startTime.toString());
         params.append('end', endTime.toString());
         params.append('step', resolution.toString());
-        // TODO path prefix here and elsewhere.
         break;
       case 'table':
-        path = '/api/v1/query';
+        path = `query`;
         params.append('time', endTime.toString());
         break;
       default:
         throw new Error('Invalid panel type "' + this.props.options.type + '"');
     }
 
-    fetch(`${this.props.pathPrefix}${path}?${params}`, {
+    fetch(`${this.props.pathPrefix}/${this.props.apiPath}/${path}?${params}`, {
```

Comment:
Shouldn't this just use the constant for the API path now and not pass this as a prop?